map/src/components/add-information.tsx

```diff
@@ -714,10 +714,10 @@ class AddInstructions extends React.Component<Props, State> {
             // Remove array completely if empty
             if (arr.length === 0) {
               // eslint-disable-next-line no-param-reassign
-              delete (info.contact[type] as ContactDetails)[method];
```

Comment:
where did this come from?

Reply:
its more like a lint:fix or prettier thing than anything else

Reply:
not sure what this is from.  this file should not have been changed by your push.  please start again by forking the repo first.   set your upstream to reach4help/reach4help.  always do git merge upstream/development  before you push.